tasks/drupal/install_from_drush.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Remove settings link
+  file:
+    path: "{{ steamengine_project_root_path_web }}/web/sites/default/settings.php"
+    state: absent
+  when: not steamengine_drupal_install_use_existing_config
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Remove settings
+  file:
+    path: "{{ steamengine_conf_path }}/settings.php"
+    state: absent
+  when: not steamengine_drupal_install_use_existing_config
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Install Drupal with drush.
+  command: >
+    {{ steamengine_drush_launcher_path }} -r {{ steamengine_project_root_path_web }}
+    site-install {{ steamengine_drupal_install_profile }} -y {{ steamengine_drupal_install_option }}
+    {{ steamengine_drupal_install_use_existing_config | ternary('--existing-config', '') }}
+  when: new_deploy and steamengine_drupal_install_from_drush
+  tags:
+    - steamengine_deploy_drupal
+
+- name: "Move setting files to {{ steamengine_conf_path }}"
+  command: "mv {{ steamengine_project_root_path_web }}/web/sites/default/settings.php {{ steamengine_conf_path }}/settings.php"
+  when: not steamengine_drupal_install_use_existing_config
+  tags:
+    - steamengine_deploy_drupal
+
+- name: Create configuration link
```

Comment:
Crash if settings.php is provided in the build.

Reply:
Il est supprimé au début du fichier, je vois pas comment c'est possible qu'il soit encore présent à ce niveau

Reply:
Yep. My comment is on the wrong file. Crash appears in deploy.yml on the same task.